react/README.md

```diff
@@ -30,6 +30,11 @@ This course is aimed to the students of the RS School which passed RS School sta
 
 [Task 1](modules//tasks/class-components.md)
 
+#### Tests
+
+- Test 1. React components
+- Test 2. React error boundary
```

Comment:
Wouldn't we want to make them links to modules so it would be like in stage 2?
![image](https://github.com/rolling-scopes-school/tasks/assets/69504277/1c48a200-ba27-4a6e-ae42-fd6b4e181468)

Reply:
I am not sure about this because I was doing like Stage 1.
![image](https://github.com/rolling-scopes-school/tasks/assets/78261193/959c237f-dab0-4790-a262-d72e0645c009)